.travis.yml

```diff
@@ -5,6 +5,7 @@ install:
 
 env:
    - TOXENV=pep8
+   - TOXENV=yamllint
```

Comment:
It was named according to the name of the linter `yamllint` and already merged in several repositories.